src/Movie.elm

```diff
@@ -77,27 +134,91 @@ movieCard selectedGenres movie =
                 _ ->
                     Set.size (Set.intersect movie.genres selectedGenres) == 0
     in
-        a
+        button
             [ classList
                 [ ( Style.MovieCard, True )
                 , ( Style.Filterable, True )
                 , ( Style.Filtered, filtered )
                 ]
-            , href movie.url
-            , target "_blank"
+            , Selected movie |> focusMovie |> onClick
```

Comment:
Prety minor, but I think this reads clearer as `onClick <| focusMovie <| Selected movie`

I suspect it could also be implemented as just `onClick <| focusMovie movie`, where the `Selected` part is handled in `update`